cmd/createenvironment.go

```diff
@@ -38,7 +38,7 @@ var environmentCmd = &cobra.Command{
 	Short: "Create a new environment",
 	Long: AddAppName(`Create a new environment
     Example:
-    $AppName create environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath.json`),
+    $AppName create environment env1 --organization org --file ./somepath.json --artifacts ./artifactsfolder`),
```

Comment:
Can you also fix this to something more meaningful and ending with yaml.
Like organizationdefinition.yaml